src/layouts/MainPage.js

```diff
@@ -0,0 +1,41 @@
+import { Box, Container, Grid } from '@mui/material'
+import { AppBar } from '../components/AppBar'
+import AdItem from '../components/advertisement/AdItem'
+import { CategoryItem } from '../components/CategoryItem'
+import ads from '../resources/ads.json'
+
+const categories = [
+  { title: 'املاک', value: 'real-estate' },
+  { title: 'املاک', value: 'real-estate' },
+  { title: 'املاک', value: 'real-estate' },
+  { title: 'املاک', value: 'real-estate' },
+  { title: 'املاک', value: 'real-estate' },
+  { title: 'املاک', value: 'real-estate' },
+]
```

Comment:
??

Reply:
temporarily created for test.
after adding the API calls will be removed